app/controllers/hyrax/dashboard/collection_members_controller.rb

```diff
@@ -0,0 +1,65 @@
+module Hyrax
+  module Dashboard
+    ## Shows a list of all collections to the admins
+    class CollectionMembersController < Hyrax::My::CollectionsController
+      before_action :filter_docs_with_read_access!
+
+      include Hyrax::Collections::AcceptsBatches
+
+      def after_update
+        respond_to do |format|
+          format.html { redirect_to success_return_path, notice: 'Collection was successfully updated.' }
```

Comment:
I believe there should be an internationalization of the notice.

Reply:
That was a copy and paste error.  So I fixed it in the new controller and in the collections controller that was the source of the error.